components/resolved_ts/src/resolver.rs

```diff
@@ -710,4 +859,31 @@ mod tests {
         assert_eq!(resolver.locks_by_key.len(), 0);
         assert_eq!(resolver.lock_ts_heap.len(), 0);
     }
+
+    #[test]
+    fn test_large_txn_tracking() {
+        let memory_quota = Arc::new(MemoryQuota::new(std::usize::MAX));
+        let txn_status_cache = Arc::new(TxnStatusCache::new(100));
+        let mut resolver = Resolver::new(1, memory_quota, txn_status_cache.clone());
+        let key: Vec<u8> = vec![1, 2, 3, 4];
+
+        // track a large txn lock
+        resolver.track_lock(1.into(), key.clone(), None, 1).unwrap();
+        assert_eq!(resolver.num_locks(), 1);
+        assert_eq!(resolver.num_transactions(), 1);
+        assert_eq!(resolver.locks_by_key.len(), 0);
+        assert_eq!(resolver.large_txn_ts.len(), 1);
+        assert_eq!(resolver.large_txn_key_representative.len(), 1);
+        assert_eq!(resolver.resolved_ts(), TimeStamp::zero());
+
```

Comment:
May also test the large txn who has the min `start_ts` without min `min_commit_ts`.

```
txn1: start_ts: 1, min_commit_ts: 10
txn2: start_ts: 2, min_commit_ts: 5
=>
resolve_ts == 5
```